### PR TITLE
Pass errors to express

### PR DESCRIPTION
**Pull request by lilsweetcaligula**
#1

### CODE REVIEW

**Comment by rjrodger** [comment]
this needs to be optional - controlled by a plugin option

**Comment by lilsweetcaligula** [comment]
> this needs to be optional - controlled by a plugin option

Commit: https://github.com/senecajs/seneca-gateway-express/pull/2/commits/face1628a2f25ef4fd460f78fe889c90bcae5423

File: src/gateway-express.ts [hunk 1]
```diff
@@ -14,19 +14,41 @@ function gateway_express(this: any, options: any) {
...
+        //
+        // This is useful, for example, when you want to return an HTTP
+        // 404 for the 'act_not_found' error; - or handle any other error
+        // that you defined and threw inside a Seneca instance.
+        //
+        return next(out)
       }
+
+      return res.send(out)
+    } catch (err) {
```
Comment (rjrodger) [inline src/gateway-express.ts:44]: isn't this redundant? catch and rethrow is the same as no catch

Comment (lilsweetcaligula) [reply src/gateway-express.ts:44]: @rjrodger I introduced the construct deliberately in order to communicate to the reader that if the `gateway` function call throws, then we do not handle that error. Omitting the construct could erroneously lead the reader into thinking there was a bug.

**Pull request merged**